Gemfile

```diff
@@ -1,8 +1,8 @@
 source 'https://rubygems.org'
 
-BUNDLER_VERSION = '1.14.6'.freeze
+BUNDLER_VERSION = '1.13.7'.freeze
```

Comment:
heroku is on a lower version of bundler.... we could create a custom buildback for it, but it seems like it'd be more trouble than it's worth.   The older bundler version doesn't seem to affect anything with the app.